elooplib/tp_tasks.hpp

```diff
@@ -12,7 +12,7 @@ class FileITask : public ITask
   FileITask(const std::string &, int, std::atomic<int> *, std::atomic<int> *, int);
   int handleFile(const std::string &, const std::string &,
                  std::function<int(FILE *, FILE *, int)> &);
-  virtual int doWork(){};
+  virtual int doWork(){return 0;};
```

Comment:
I guess this should be pure virtual. BTW why does this class return int instead of void?